### ISSUE TITLE
I am interested in COLO

### ISSUE THREAD

**Issue by hongkongpost** [wangchenghku/COLO#1]
but when i test the COLO https://wiki.xenproject.org/wiki/COLO_-_Coarse_Grain_Lock_Stepping
it is hard to follow the steps, eg where is the path of guest config
could you suggest any material to let me know more about COLO?
and and forum is able to let me ask some questions?

**Comment by wangchenghku** [comment]
Hi, I have never set up COLO on Xen. This repo is the tutorial for setting up COLO on KVM. I think you can contact the developers by email or leave message in QEMU forum. For your reference: https://wiki.qemu.org/Features/COLO
